at-hoangtran/Week4/Day1/Exercise-Day1.rb

```diff
@@ -0,0 +1,87 @@
+module NewsModule
+  attr_accessor :id, :title, :content
+  def update(news, title, content)
+    self.title = title
+    self.content = content
+  end
+
+  def delete(news)
+    @list_news.delete(news)
+  end
+end
+class BbcNews
+  extend NewsModule
+  @list_news = []
+  def initialize
+    @id = id
+    @title = title
+    @content = content
+    @category = category
+  end
+
+  def self.new(id, title, content, category)
+    h = {}
+    h[id] = [title, content, category]
+    @list_news.push(h)
+  end
+end
+
+class CnnNews
+  extend NewsModule
+  @list_news = []
+  def initialize
+    @id = id
+    @title = title
+    @content = content
+    @publication_date = publication_date
+  end
+
+  def self.new(id, title, content, publication_date)
+    h = {}
+    h[id] = [title, content, publication_date]
+    @list_news.push(h)
+  end
+end
+
+class User
+  attr_accessor :id, :name, :message, :friends, :subcription_list
+  def initialize(id, name)
+    @id = id
+    @name = name
+    @message = message
+    @friends = friends
+    @subcription_list = subcription_list
+  end
+
+  def subscribe(organisation)
+    @subcription_list.push(organisation)
+  end
+
+  def unsubcribe(organisation)
+    if @subcription_list.include?(organisation)
+      @subcription_list.delete(organisation)
+    else
+      false
+    end
+  end
+
+  def add_friends(user)
+    @friends[user.id] = user.name
+    user.friends[self.id] = self.name
+  end
+
+  def shows_news(organisation)
+    if @organisation.include?(organisation)
+      @organisation.each do |x|
+        puts x.list if x == organisation
+      end
+    else
+      nil
+    end
+  end
+end
+
+bbc_new_01 = BbcNews.new(0o1, 'Bbc First News', 'Hello world', 'Say Hello')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/3b12e820232834cdf65fef2d6d21b32dd53b4102/at-hoangtran/Week4/Day1/Exercise-Day1.rb#L84"></span>Naming/VariableNumber: Use normalcase for variable numbers.</td>
    </tr>
  </tbody>
</table>